notebooks/nessie-delta-demo-nba.ipynb

```diff
@@ -0,0 +1,642 @@
+    "\n",
+    "- Create a new branch named `dev`\n",
+    "- List all branches\n",
+    "\n",
+    "It is worth mentioning that we don't have to explicitly create a `main` branch, since it's the default branch."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "spark.sql(\"CREATE BRANCH dev AS main\").toPandas()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We have created the branch `dev` and we can see the branch with the Nessie `hash` its currently pointing to. \n",
+    "\n",
+    "Below we list all branches. Note that the auto created `main` branch already exists and both branches point at the same `hash`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "spark.sql(\"LIST REFERENCES\").toPandas() "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create tables under dev branch\n",
+    "-------------------------------------\n",
+    "Once we created the `dev` branch and verified that it exists, we can create some tables and add some data.\n",
+    "\n",
+    "We create two tables under the `dev` branch:\n",
+    "- `salaries`\n",
+    "- `totals_stats`\n",
+    "\n",
+    "These tables list the salaries per player per year and their stats per year.\n",
+    "\n",
+    "To create the data we:\n",
+    "\n",
+    "1. switch our branch context to dev\n",
+    "2. create the table\n",
+    "3. insert the data from an existing csv file. This csv file is already stored locally on the demo machine. A production use case would likely take feeds from official data sources"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "spark.sql(\"USE REFERENCE dev\")\n",
+    "\n",
+    "# Creating `salaries` table\n",
+    "spark.sql(\"CREATE TABLE IF NOT EXISTS salaries (Season STRING, Team STRING, Salary STRING, Player STRING) USING delta LOCATION '{}/salaries'\".format(warehouse))\n",
+    "\n",
+    "spark.sql('CREATE OR REPLACE TEMPORARY VIEW salaries_table USING csv OPTIONS (path \"../datasets/nba/salaries.csv\", header true)')\n",
+    "spark.sql('INSERT INTO salaries SELECT * FROM salaries_table')\n",
+    "\n",
+    "# Creating `totals_stats` table\n",
+    "spark.sql(\"\"\"CREATE TABLE IF NOT EXISTS totals_stats (\n",
+    "             Season STRING, Age STRING, Team STRING, ORB STRING, DRB STRING, TRB STRING, AST STRING, STL STRING, \n",
+    "             BLK STRING, TOV STRING, PTS STRING, Player STRING, RSorPO STRING) \n",
+    "             USING delta LOCATION '{}/total_stats'\"\"\".format(warehouse))\n",
+    "spark.sql('CREATE OR REPLACE TEMPORARY VIEW stats_table USING csv OPTIONS (path \"../datasets/nba/totals_stats.csv\", header true)')\n",
```

Comment:
Should be line-wrapped (horizontal scrolling)

Reply:
done